arch/Kconfig

```
@@ -1158,6 +1158,13 @@ config ARCH_MINIMAL_VECTORTABLE
 		IRQMAPPED_MAX, then hardware interrupt vector 42 is not used and
 		if it occurs will result in an unexpected interrupt crash.
 
+config ARCH_MINIMAL_VECTORTABLE_DYNAMINC
```

Typo. Please check other occurrences.

```suggestion
config ARCH_MINIMAL_VECTORTABLE_DYNAMIC
```